src/Controller/ControllerManager.php

```diff
@@ -73,10 +74,10 @@ public function injectControllerDependencies($controller, ServiceLocatorInterfac
             // is why the shared EM injection needs to happen; the conditional
             // will always pass.
             $events = $controller->getEventManager();
-            if (!$events instanceof EventManagerInterface) {
+            if (! $events instanceof EventManagerInterface
```

Comment:
Yes it does! It inherits from EventsCapableInterface, which defines the
getter.
On Oct 1, 2015 6:37 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In src/Controller/ControllerManager.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40903723>:
>
> > @@ -73,10 +74,10 @@ public function injectControllerDependencies($controller, ServiceLocatorInterfac
> >              // is why the shared EM injection needs to happen; the conditional
> >              // will always pass.
> >              $events = $controller->getEventManager();
> > -            if (!$events instanceof EventManagerInterface) {
> > +            if (! $events instanceof EventManagerInterface
>
> Urgh, wait, the interface defines no getters...
> https://github.com/zendframework/zend-eventmanager/blob/a6c747fe59658fb62feb23613d7ea22146b6b611/src/EventManagerAwareInterface.php#L15-L24
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40903723>.
>